packages/ice-plugin-dll/lib/config/webpack.dll.js

```diff
@@ -0,0 +1,32 @@
+const webpack = require('webpack');
+const path = require('path');
+
+module.exports = (rootDir) => {
+  const join = path.join.bind(path, rootDir);
+
+  return {
+    entry: {
+      vendor: [
+        join('node_modules', 'plugin-dll', 'vendor.js'),
+      ],
+    },
+    output: {
+      path: join('public'),
```

Comment:
We only need to take care of the public directory because files will be copied over to the `/build` folder [here](https://github.com/ice-lab/ice-scripts/blob/master/packages/ice-scripts/lib/config/setWebpackPlugins.js#L50-L57).